src/Components/Card/styles.css

```diff
@@ -0,0 +1,13 @@
+.btn-primary {
+  color: white;
```

Comment:
you can use variable naming here

Reply:
Only for black and white colors, I use this otherwise I use variable naming